src/importer/onnx/onnx_importer.cpp

```diff
@@ -788,3 +788,11 @@ shape_t onnx_importer::broadcast_shape(const shape_t &v_shape, const shape_t &in
 
     return result;
 }
+
+std::string onnx_importer::generate_name(const onnx::NodeProto &node) const
+{
+	if (node.has_name())
```

Comment:
Use spaces instead of tabs.

Reply:
Oh, sorry for that. Indeed I pay special attention to fixing things like this, but missed this one. Thank you, will add clang-format to my pipeline to have it fixed automatically.